gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/service/MinioHandler.java

```diff
@@ -117,6 +128,10 @@ public void updateTasks(Event event) {
         }
     }
 
+    boolean isTasksRunningOrPending(Set<Task> tasks) {
```

Comment:
Naming: this method handles a Collection of Tasks but the method's name does not make this clear (the "s" at the end of "Tasks" is easily forgotten when you read the method's name).
As the method returns true if any task of the Collection is running or pending, I suggest changing the method's name to something like: `isAnyTaskRunningOrPending`.